Pods/Charts/LICENSE

```diff
@@ -0,0 +1,202 @@
+Apache License
```

Comment:
Common practice says to ignore Pods/ folder from git tracking. Read more about `.gitignore` file and check the https://www.gitignore.io to find the default `.gitignore` file for Swift development.

I would like also to add that on some projects Pods including is a good practice. But it should be a strong decision over the common practice.